Dockerfiles/midstream/Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM registry.fedoraproject.org/fedora:33
-WORKDIR /project
 ARG OPERATOR_SDK_VERSION=v1.4.0
-ARG OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
 ARG UMOCI_VERSION=v0.4.5
-ADD ./fix_etc_passwd.sh /usr/bin/fix_etc_passwd.sh
+ENV ANSIBLE_CONFIG=/operator-test-playbooks/Dockerfiles/midstream/
+ENV ANSIBLE_LOCAL_TEMP=/tmp/
+ENV PATH="/operator-test-playbooks/Dockerfiles/midstream/:${PATH}"
```

Comment:
```
+ADD ./Dockerfiles/midstream/run_tests.py ./Dockerfiles/midstream/unit_tests.py ./Dockerfiles/midstream/fix_etc_passwd.sh /usr//bin/
+ADD roles /project/operator-test-playbooks/roles/
+ADD extract-operator-bundle.yml validate-operator-bundle.yml /project/operator-test-playbooks/
+CMD ["/usr/bin/run_tests.py"]
```
I agree with the binaries. I can change it to reflect the same. 
However, the CMD is no longer recommended inside the container because. It would not work for the midstream team. They would like to have the fix_etc_password.sh as it is within the image.

Reply:
line is not needed anymove; cause executable scripts are in right place
```
ADD ./Dockerfiles/midstream/run_tests.py ./Dockerfiles/midstream/unit_tests.py ./Dockerfiles/midstream/fix_etc_passwd.sh /usr/bin/
```